api/v1alpha08/kogitoserverlessbuild_types.go

```diff
@@ -15,38 +15,96 @@
 package v1alpha08
 
 import (
+	"encoding/json"
+
+	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
 
-	"github.com/kiegroup/kogito-serverless-operator/container-builder/api"
+type BuildPhase string
+
+const (
+	// BuildPhaseNone --
+	BuildPhaseNone BuildPhase = ""
```

Comment:
isn't better to use None instead blank string?

Reply:
Nops, because this is the null pattern that we use to bind to the inner build. The inner build might not be ready, hence an empty phase.